entrypoint.sh

```diff
@@ -1,14 +1,26 @@
 #!/bin/sh
 
-echo "Collecting static files"
+set -o pipefail -o nounset
+
+
+echo >&2 "Collecting static files."
 rm -rf /usr/src/static/*
-python manage.py collectstatic
+python3 manage.py collectstatic
 
-echo "Waiting for postgres..."
+echo >&2 "Waiting for postgres..."
 while ! nc -z db 5432; do
-  sleep 0.1
+	sleep 0.1
 done
-echo "PostgreSQL started"
-python manage.py migrate
+echo >&2 "PostgreSQL started."
+
+echo >&2 "Migrating database."
+python3 manage.py migrate
+
+echo >&2 "Creating admin user (username: admin, password: admin)."
+export DJANGO_SUPERUSER_EMAIL=admin@admin.admin
+export DJANGO_SUPERUSER_PASSWORD=admin \
+export DJANGO_SUPERUSER_USERNAME=admin
+# This will fail if admin user already exists.
+python3 manage.py createsuperuser --noinput || true
```

Comment:
What problem do you have in mind? Can you elaborate?

Reply:
It becomes impossible to remove/change username of the default user, as it will get recreated when the container restarts. If you really want automatic user creation, it should only occur once, when the database is first created.

But personally I don't think this is necessary at all... If you want a development setup, call `populate_with_test_data` after starting the container which creates the admin account too iirc, if you want a production setup, don't use `admin` as the password...